Cslotboom updates

cslotboom opened the pull request:
I've added the readOBD function. I've also added a few tests that work with pytest, including a generic test for the readOBD function.
I changed the "saveNodesandElements" and "readNodesandElements" to use a space instead of a comma as a delimiter.

The intention for this pull request is to be on u-anurag's plotting branch. If anyone else is seeing this then I have made a mistake...

cslotboom committed:
Create test_plot_model_2D.py

cslotboom committed:
Create test_plot_model_3D.py

cslotboom committed:
Merge branch 'OpenSeesPy-Plotting' of https://github.com/u-anurag/OpenSeesPy into OpenSeesPy-Plotting

cslotboom committed:
readOBD + tests

Added a readOBD command, and tests for the read OBD command.

u-anurag commented on openseespy-pip/openseespy/postprocessing/Get_Rendering.py:
```diff
@@ -78,7 +78,7 @@ def createODB(*argv):
 	
 	# Save node and element data in the main Output folder
-	idbf.saveNodesandElements(ODBdir)
+	idbf.saveNodesandElements(ModelName, LoadCaseName)
```
This input argument should be ODBdir since nodes and element data does not change with a load case. It should be saved in the main ODB folder and not in the load case-specific folder.

cslotboom replied:
Sounds good. This will require that I modify saveNodesandElements to save in the ODBdir. I'll make the change and update.

u-anurag commented on openseespy-pip/openseespy/postprocessing/Get_Rendering.py:
```diff
@@ -104,7 +104,64 @@ def createODB(*argv):
+        pass
+	
+	# Read node and element data in the main Output folder
+    nodes, elements = idbf.readNodesandElements(ModelName, LoadCaseName)
```
The input argument should be ODBdir as explained earlier. 

nodes, elements = idbf.readNodesandElements(ODBdir)

u-anurag reviewed (changes requested):
@cslotboom Check the comments in the review. The PR can be merged after these are settled.

cslotboom committed:
Changed the saveNodesandElements

Functions no longer take argv. values. Functions read from the ODB location, instead of the load case location

u-anurag reviewed (commented):
@cslotboom Just remembered, should we change these internal commands to _saveNodesandElements(ODBdir) and _readNodesandElements(ODBdir) instead, to follow the plotting development guide? All saving and reading of data will be through createODB() and readODB() user commands.

cslotboom committed:
made readNodesandElements Internal

made readNodesandElements Internal by adding a '_' to all instaces of the name.
